test/testcafe/framework/page-objects/DeviceCodeActivatePageObject.js

```diff
@@ -9,10 +10,16 @@ export default class DeviceCodeActivatePageObject extends BasePageObject {
   }
 
   getPageSubtitle() {
+    if (userVariables.v3) {
+      return this.form.getElement('[data-se="o-form-explain"]').textContent;
+    }
     return this.form.getElement('.okta-form-subtitle').textContent;
   }
 
-  clickNextButton() {
+  clickNextButton(name) {
+    if (userVariables.v3) {
+      return this.form.clickSaveButton(name);
+    }
     return this.form.clickSaveButton();
```

Comment:
I think this would work.
```suggestion
    return this.form.clickSaveButton(name);
```